.github/workflows/submit.yml

```diff
@@ -8,13 +8,11 @@ jobs:
   submit:
     uses: curvenote/actions/.github/workflows/submit.yml@v1
     with:
-      id-pattern-regex: '^morganton-2025-(?:[a-zA-Z0-9-_]{3,15})$'
+      id-pattern-regex: '^morganton-2025-(?:[a-zA-Z0-9-_]{2,15})$'
       venue: ncssm-mor
       collection: '2025'
       kind: original
       path: papers/*
-      label: publish
```

Comment:
`label` only works for actions on `pull_request`; for `push` there are no labels to check.